docs/site/HasMany-relation.md

```diff
@@ -416,6 +416,20 @@ factory `orders` for instances of `customerRepository`:
   instance
   ([API Docs](https://loopback.io/doc/en/lb4/apidocs.repository.hasmanyrepository.patch.html))
 
+Notice that navigational properties can not be included in the data for CRUD
```

Comment:
I'm thinking whether it's better to reword it as follows (i.e. to begin with what's the right way to do it :) ):


For creating the related models, you can:
```ts
const myCustomer = await customerRepository.create({id: 1, name: 'Fiorio'});
const orderData = {id: 1, customerId: myCustomer.id};
// create the related order
customerRepository.orders(myCustomer.id).create(orderData);
```

Since `CustomerRepository.create()` expects a `Customer` model only, if the related model is included in the argument, it will result in `Navigational properties are not allowed` error. That means, the following code snippet is not valid:
```
customerRepository.create({
      id: 1,
      name: 'invalid request',
      orders: [{id: 1, customerId: 1}],
});
```

Reply:
👍 Agreed. I made a bit changes, please check. Thanks!